.github/workflows/pull_requests.yml

```diff
@@ -0,0 +1,65 @@
+# CI stages to execute against Pull Requests
+name: Nailgun - CI
+
+on:
+  pull_request:
+    types: ["opened", "synchronize", "reopened"]
+
+env:
+    PYCURL_SSL_LIBRARY: openssl
+
+jobs:
+  codechecks:
+    name: Code Quality
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.8, 3.9]
```

Comment:
same suggestion about 3.10

Reply:
Removed `3.9` and added `3.10` which now checks in even python versions of 6,8 and 10.